Starting Location fixes for Special Assignments.

dribbleondo opened the pull request:
My bad. I mean, I dislike these discount-permanent Elusive Targets, hence my week-long absence.

dribbleondo committed:
fixed locations in the new contracts.

finally got round to playing them. Jesus christ they are lazy.

dribbleondo committed:
removed three starting locations from Special Assignments.

(Chawls, Photoshoot and Shaman's Hut). Also Photoshoot is one work Kotti, not two =P.

dribbleondo committed:
Missed one. Bus stop removed.

Bus stop removed.

dribbleondo committed:
changelog update...again.

four locations removed.

TheKotti commented on about.html:
```diff
@@ -68,6 +68,14 @@ <h3 style="display:inline">Changelog</h3>
         Show
       </button>
       <ul id="changelog" style="display:none">
+        <li class="versionnumber">0.532.5</li>
```
Remove this whole thing

dribbleondo replied:
done.

TheKotti commented on about.html:
```diff
@@ -68,6 +68,14 @@ <h3 style="display:inline">Changelog</h3>
+        </li>
+        <li class="versionnumber">0.532</li>
+        <li>
+          Fixed four incorrect starting locations in the Permanant Elusi- er.."Special Assignments" (If you're curious, it was the "Shaman's Hut" and "Village Bus Stop" starting points in Columbia, and the "Chawls" and "Photoshoot" starting points in Mumbai)
```
Just call them special assignments, no need to be a smartass here.

dribbleondo replied:
doneso.

TheKotti reviewed (changes requested):
Would strongly recommend looking into rebasing so there's not 4 commits over a couple of json entries.

dribbleondo committed:
removed humour.

as per request, removed the 532.5 from the changelog. Gonna rebase it after pushing this.

dribbleondo commented:
I've tried rebasing it, and Github won't update the pull requests as such. I'm just gonna close this commit, delete the branch I have, re-fork your branch, then add the commits that way. A very annoying solution.

I'm probably doing something wrong.
